### ISSUE TITLE
[FEATURE] Capitulos/chapters

### ISSUE THREAD

**Issue by NoobSam777** [mosfete/front-end-flataforma-cursos#1]
Seria possível implementar o sistema de capítulos como no YouTube e telegram?
![image](https://github.com/mosfete/front-end-flataforma-cursos/assets/49501697/c05b56ae-a8fc-49d9-97ff-08c319642b82)

Algumas receitas são vídeos longos. Porém, são separados por várias partes para uma melhor navegação

**Comment by dragaoinvisivel** [comment]
Sim, é possível. No momento não tenho tempo para implementar, então se ninguém tomar a iniciativa até ter tempo eu dou inicio a adição dessa nova funcionalidade

**Comment by diego-humberto** [comment]
Eu tentei modificar aqui, depois tenta conferir se está bom.
